spec/system/page_objects/pages/admin_flags.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class AdminFlags < PageObjects::Pages::Base
+      def toggle(key)
+        PageObjects::Components::DToggleSwitch.new(".#{key}").toggle
```

Comment:
Maybe scope this a bit further e.g. `.admin-flags-row.#{key}` or however the class is structured